src/lib.rs

```diff
@@ -94,61 +104,61 @@ impl AuthConfig {
 
 #[cfg(test)]
 mod tests {
-    #[ignore = "dead_code"]
     use super::*;
     use std::fs;
 
     #[tokio::test]
     async fn test_generate_auth_token() {
-        let config = get_valid_jwt();
-        let token = config.generate_auth_token(3600).await;
-        assert!(token.is_ok());
-        println!("{}", token.unwrap());
+        let valid_config = get_valid_config_complete();
+
+        // The following config depends on an deleted service account key.
+        // It is no longer possible to create tokens with this info.
```

Comment:
thaht my be a comment you want to write in an #[ignore = "..."] attribute to get the test ignored?